README.md

```diff
@@ -1,77 +1,85 @@
-![](https://github.com/nomad-coe/nomad-schema-plugin-example/actions/workflows/actions.yml/badge.svg)
```

Comment:
This should be switched to the workflow of this repo but would be good to include.